openapi/components/schemas/LegacySubscription.yaml

```
@@ -0,0 +1,482 @@
+      deprecated: true
+      properties:
+        type:
+          description: Type of invoice line item.
+          type: string
+          enum:
+            - debit
+            - credit
+        description:
+          description: Description of the line item.
+          type: string
+          maxLength: 1000
+        unitPriceAmount:
+          description: Unit price of the line item.
+          type: number
+          format: double
+          example: 49.95
+        unitPriceCurrency:
+          $ref: ./CurrencyCode.yaml
+        quantity:
+          description: Quantity of the line item.
+          type: integer
+          example: 1
+        periodStartTime:
+          description: Date and time when the service period begins for this item.
+          type: string
+          format: date-time
+        periodEndTime:
+          description: Date and time when the service period ends for this item.
+          type: string
+          format: date-time
+        createdTime:
+          $ref: ./CreatedTime.yaml
+        updatedTime:
+          $ref: ./UpdatedTime.yaml
+  lineItemSubtotal:
+    type: object
+    readOnly: true
+    description: |-
+      Subtotal of line items in this subscription (signed value).
+      If credits exceed debits, this value is a negative number.
+    properties:
+      currency:
+        $ref: ./CurrencyCode.yaml
+      amount:
+        type: number
+        x-type: Money
+        x-sortable: true
+        description: Amount of the subtotal.
+        format: double
+        example: 49.95
+  paymentInstrumentId:
+    type:
+      - 'string'
+      - 'null'
+    description: |-
+      ID of the payment instrument to use for autopay.
+      If this value is not provided, or if the payment instrument is inactive,
+      the customer's default payment instrument is used.
+    maxLength: 50
+    example: inst_0YVB8KPKNXCBR9EDX7JHSED75N
+  billingStatus:
+    description: |-
+      Billing status of the most recent invoice.
+      This value may help you to determine if you should change the service status of the service,
+      such as suspending the service.
+    type: string
+    readOnly: true
+    enum:
+      - draft
+      - unpaid
+      - past-due
+      - abandoned
+      - paid
+      - voided
+      - refunded
+      - disputed
+      - partially-refunded
+      - partially-paid
+  websiteId:
+    x-sortable: true
+    x-basic: true
+    allOf:
+      - $ref: ./WebsiteId.yaml
+  currency:
+    description: Currency of the order.
+    readOnly: true
+    $ref: ./CurrencyCode.yaml
+  initialInvoiceId:
+    description: ID of the initial invoice.
+    readOnly: true
+    type:
+      - 'string'
+      - 'null'
+    maxLength: 50
+    example: in_0YVF9605RKC62BP14NE2R7V2XT
+  recentInvoiceId:
+    description: |-
+      ID of the most recently issued invoice.
+      The invoice might not be `paid` yet.
```

suggestion: for readability
I'm not sure what the second sentence means in relation to the first. I've made a suggestion based on my understanding, which may not be correct
```suggestion
      This value is `null` if no invoice is `paid` yet.
```